source/includes/User/role.md

```diff
@@ -14,76 +14,307 @@ A **Role** represents a group of privileges in the system. Roles may inherit pri
   "description": "Manages forms and attaches them to the UI"
 }
+
+```
+
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{\r\n  \"description\": \"Manages forms and attaches them to the UI\"\r\n}\r\n");
+Request request = new Request.Builder()
+  .url("https://demo.openmrs.org/openmrs/ws/rest/v1/role/86f043da-1019-4caf-8db6-d5a472334f58")
+  .method("POST", body)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Content-Type", "application/json")
+  .addHeader("Cookie", "JSESSIONID=955E778A4F700C5AA9651DAF6FC9DDDA")
+  .build();
+Response response = client.newCall(request).execute();
+
+```
+
+```javascript
+
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Content-Type", "application/json");
+myHeaders.append("Cookie", "JSESSIONID=955E778A4F700C5AA9651DAF6FC9DDDA");
+
+var raw = JSON.stringify({"description":"Manages forms and attaches them to the UI"});
+
+var requestOptions = {
+  method: 'POST',
+  headers: myHeaders,
+  body: raw,
+  redirect: 'follow'
+};
+
+fetch("https://demo.openmrs.org/openmrs/ws/rest/v1/role/86f043da-1019-4caf-8db6-d5a472334f58", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+
+
 ```
 
 *  Update a role with given UUID, this method only modifies properties in the request. Returns a `404 Not Found` status if the role does not exist. If the user is not logged in to perform this action, a `401 Unauthorized` status is returned.
 
-    ### Attributes
+### Attributes
 
     Parameter | Type | Description
     --- | --- | ---
     *name* | `String` | Name of the role (Required)
     *description* | `String` | Description of the role (Required)
-    *privileges* | `Array[] : privileges` | An array of the privilege resource
-    *inheritedRoles* | `Array[] : inheritedRoles` | An array of the inheritedRoles type
+    *[privileges](#privilege)* | `Array[] : privileges` | An array of the privilege resource
+    *[inheritedRoles](#role)* | `Array[] : inheritedRoles` | An array of the inheritedRoles type or UUIDs
 
 
 ## Delete a role
 
-```console
+> Delete a role
+
+```shell
 DELETE /role/:target_role_uuid?purge=true
 ```
 
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("text/plain");
+RequestBody body = RequestBody.create(mediaType, "");
+Request request = new Request.Builder()
+  .url("https://demo.openmrs.org/openmrs/ws/rest/v1/role/86f043da-1019-4caf-8db6-d5a472334f58?purge=true")
+  .method("DELETE", body)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Cookie", "JSESSIONID=955E778A4F700C5AA9651DAF6FC9DDDA")
+  .build();
+Response response = client.newCall(request).execute();
+
+```
+
+```javascript
+
+var myHeaders = new Headers();
```

Comment:
@LuGO0  can you change the name my_header to  a one sensible

Reply:
ok @Akayeshmantha should I make it requestHeader will it be apt?

Reply:
I will make changes in all the examples if this looks good 👍

Reply:
it looks good no need to change on the already  committed once please change on existing PR's